client/components/Navbar.js

```diff
@@ -1,47 +1,156 @@
-import React from 'react'
-import {connect} from 'react-redux'
-import {Link} from 'react-router-dom'
-import {logout} from '../store'
+import React from "react";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import { logout } from "../store";
 
-const Navbar = ({handleClick, isLoggedIn}) => (
+const Navbar = ({ handleClick, isLoggedIn }) => (
   <div>
-    <h1>FS-App-Template</h1>
-    <nav>
-      {isLoggedIn ? (
-        <div>
-          {/* The navbar will show these links after you log in */}
-          <Link to="/home">Home</Link>
-          <a href="#" onClick={handleClick}>
-            Logout
-          </a>
-        </div>
-      ) : (
-        <div>
-          {/* The navbar will show these links before you log in */}
-          <Link to="/login">Login</Link>
-          <Link to="/signup">Sign Up</Link>
-        </div>
-      )}
-    </nav>
-    <hr />
+    <div className="navBar">
+      <nav>
+        <ul id="main" className="navbar-left">
+          <li>
+            <span>Shop </span>
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="16"
+              height="16"
+              fill="currentColor"
+              className="bi bi-chevron-down"
+              viewBox="-2 -3 16 16"
+            >
+              <path
+                fillRule="evenodd"
+                d="M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z"
+              />
+            </svg>
+            <ul className="drop">
+              <div>
+                <li>
+                  <Link to="/wine">Wine</Link>
+                </li>
+                <li>
+                  <Link to="/beer">Beer</Link>
+                </li>
+              </div>
```

Comment:
also to make it smaller, id do this instead 
```suggestion
const NavLink = ({ label, beer }) =>  (
  <li><Link to={link}>{label}</Link></li>
);

<div>
  {[{ label: 'Wine', link: '/wine' }, { label: 'Beer', link: '/beer' }].map(link => <NavLink {...link} />) }
</div>


<div>
  {[{ label: 'Profile', link: '/profile' }, { label: 'Orders', link: '/orders' }].map(link => <NavLink {...link} />) }
</div>
```

Reply:
same with on lines 93-105 and 108-114